packages/cacti-plugin-copm-corda/src/main/typescript/plugin-copm-corda.ts

```diff
@@ -0,0 +1,77 @@
+import {
+  IPluginCrpcService,
+  ICactusPluginOptions,
+} from "@hyperledger/cactus-core-api";
+import {
+  Logger,
+  LoggerProvider,
+  LogLevelDesc,
+} from "@hyperledger/cactus-common";
+import { ICrpcSvcRegistration } from "@hyperledger/cactus-core-api";
+import { ServiceType } from "@bufbuild/protobuf";
+import { DefaultService } from "@hyperledger-cacti/cacti-copm-core";
+import { CopmCordaImpl, DefaultApi } from "./service-implementation";
+import { createGrpcTransport } from "@connectrpc/connect-node";
+import { createPromiseClient } from "@connectrpc/connect";
+
+export interface IPluginCopmCordaOptions extends ICactusPluginOptions {
+  logLevel?: LogLevelDesc;
+  copmKotlinServerBaseUrl: string;
+}
+
+export class PluginCopmCorda implements IPluginCrpcService {
```

Comment:
@jenniferlianne Please also implement the `ICactusPlugin` interface

Reply:
Done.